README.md

```diff
@@ -605,26 +611,28 @@ tags: |
 * [`type=semver,pattern=...`](#typesemver)
 * [`type=match,pattern=...`](#typematch)
 
-For conditionally tagging with latest for a specific branch name, e.g. if your default branch name
-is not `master`, use `type=raw` with a boolean expression:
+For conditionally tagging with latest for a specific branch name, e.g. if your
+default branch name is not `master`, use `type=raw` with a boolean expression:
 
 ```yaml
 tags: |
-  type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', github.event.repository.default_branch) }}
+  # set latest tag for master branch
+  type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', 'master') }}
 ```
```

Comment:
IMO you should remove this entirely. A solution which may not work under some circumstances should not be suggested, especially without warnings.

Reply:
I actually updated this example with a hardcoded branch name to show how to handle this case for a specific branch as it seems still valid for me.

Reply:
Oh, right. I didn't notice.